tests/backup/backup_test.go

```diff
@@ -5322,13 +5554,54 @@ func TearDownBackupRestore(bkpNamespaces []string, restoreNamespaces []string) {
 	DeleteCluster(destinationClusterName, OrgID, ctx)
 	DeleteCluster(SourceClusterName, orgID, ctx)
 	// Need to add backup location UID for Delete Backup Location call
-	DeleteBackupLocation(backupLocationName, "", OrgID)
+	err = DeleteBackupLocation(backupLocationName, "", OrgID)
+	dash.VerifySafely(err, nil, fmt.Sprintf("Deleting  backup  location %s", backupLocationName))
 	DeleteCloudCredential(CredName, OrgID, CloudCredUID)
 }
+
 func getEnv(environmentVariable string, defaultValue string) string {
 	value, present := os.LookupEnv(environmentVariable)
 	if !present {
 		value = defaultValue
 	}
 	return value
 }
+
+// ShareBackupWithMultipleUsers shares backup with multiple users with different access levels
+// It returns a map with key as userAccessContext and value as backup shared
+func ShareBackupWithMultipleUsers(backupNames []string, users []string, ctx context.Context) (map[userAccessContext]string, error) {
```

Comment:
I think a better name for this function would be `ShareBackupWithUsersAndAccessAssignment`
Let me know what do you think.

Reply:
sure.Done!!